__fixtures__/test-project/web/package.json

```diff
@@ -25,7 +25,7 @@
     "autoprefixer": "^10.4.14",
     "postcss": "^8.4.27",
     "postcss-loader": "^7.3.3",
-    "prettier-plugin-tailwindcss": "0.4.1",
+    "prettier-plugin-tailwindcss": "^0.4.1",
```

Comment:
This needs to be updated. Recommend fully re-running update test-project command.